### PR TITLE
Improve error message for all bdb docker entrypoint scripts

### PR DESCRIPTION
**Pull request by muawiakh**
- If a mandatory variable is not specified, it will exit with the relevant
code and error message.
- For more verbosity, we will also echo the values for all the mandatory
variables.

### CODE REVIEW

File: k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash [hunk 1]
```diff
@@ -7,14 +7,18 @@ MONGODB_BACKUP_CONF_FILE=/etc/mongodb-mms/backup-agent.config
...
 ca_crt_path=`printenv CA_CRT_PATH`
-backup_crt_path=`printenv BACKUP_PEM_PATH`
+backup_pem_path=`printenv BACKUP_PEM_PATH`
 
-if [[ -z "${mms_api_keyfile_path}" || \
-    -z "${ca_crt_path}" || \
-    -z "${backup_crt_path}" || \
-    -z "${mms_groupid_keyfile_path}" ]]; then
-  echo "Invalid environment settings detected. Exiting!"
+if [[ -z "${mms_api_keyfile_path:?MMS API Keyfile path not specified. Exiting!}" || \
```
Comment (krish7919) [inline k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash]: Can we use the actual env var in the error message?
`MMS API Keyfile path not specified.` => `MMS_API_KEYFILE_PATH not specified.`

Same for every other line in the PR?

File: k8s/nginx-http/container/nginx_entrypoint.bash [hunk 2]
```diff
@@ -23,17 +23,26 @@ bdb_ws_port=`printenv BIGCHAINDB_WS_PORT`
...
-  echo "Invalid environment settings detected. Exiting!"
+if [[ -z "${cluster_frontend_port:?Cluster frontend port not specified. Exiting!}" || \
+      -z "${mongo_frontend_port:?MongoDB frontend port not specified. Exiting!}" || \
+      -z "${mongo_backend_host:?MongoDB backend host not specified. Exiting!}" || \
+      -z "${mongo_backend_port:?MongoDB backend port not specified. Exiting!}" || \
+      -z "${bdb_backend_host:?BigchainDB backend host not specified. Exiting!}" || \
+      -z "${bdb_api_port:?BigchainDB API port not specified. Exiting!}" || \
+      -z "${bdb_ws_port:?BigchainDB WS port not specified. Exiting!}" || \
+      -z "${dns_server:?DNS server not specified. Exiting!}" || \
+      -z "${health_check_port:?}" ]]; then
```
Comment (krish7919) [inline k8s/nginx-http/container/nginx_entrypoint.bash]: We missed a message for the health check port.

Comment (muawiakh) [inline k8s/nginx-http/container/nginx_entrypoint.bash]: addressed

File: k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash (hunk 1 above)
Comment (muawiakh) [inline k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash]: +1, Done.

**Pull request merged**